### PR TITLE
Update delayed-delivery.md

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

**Comment by bording** [comment]
@adamralph I like all of these changes, except for the removal of the hyphens. When talking about "delay level exchanges", "delay level" is a compound adjective, so it should be hyphenated.

It looks like your addition of "delay" in front of all "level"s has introduced a third place where this is relevant.

**Comment by adamralph** [comment]
@bording OK, that makes sense. I'll hyphenate and re-hyphenate were appropriate.

**Comment by bording** [comment]
I've got a couple tweaks I want to make as well now that I've seen the page live. I'll just go ahead and push those up as commits on this PR as well.

**Comment by adamralph** [comment]
@bording 👍 to your changes.

**Comment by bording** [comment]
@Particular/docs-maintainers I think this is ready for merging.

File: nservicebus/rabbitmq/delayed-delivery.md [hunk 1]
```diff
@@ -6,31 +6,31 @@ reviewed: 2017-03-02
 versions: '[4,]'
 ---
 
-In Versions 4.3 and above, the RabbitMQ transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). The transport now creates an infrastructure inside the broker to natively handle delaying messages.
+In Versions 4.3 and above, the RabbitMQ transport no longer relies on the [timeout manager](/nservicebus/messaging/timeout-manager.md) to implement [delayed delivery](/nservicebus/messaging/delayed-delivery.md). Instead, the transport creates infrastructure inside the broker which can delay messages using native RabbitMQ features.
```
Comment (indualagarsamy) [inline nservicebus/rabbitmq/delayed-delivery.md]: Can you link it to RabbitMQ doco on native delay feature on Rabbit?

Comment (bording) [inline nservicebus/rabbitmq/delayed-delivery.md]: This is the doco, we are taking the pieces described here to create a native delayed delivery feature.

Comment (SimonCropp) [inline nservicebus/rabbitmq/delayed-delivery.md]: @bording i imagine @indualagarsamy is suggesting a link here https://www.rabbitmq.com/ttl.html

Comment (bording) [inline nservicebus/rabbitmq/delayed-delivery.md]: But that isn't directly relevant. It's just one piece of several different things that we've used to build this feature.

File: nservicebus/rabbitmq/delayed-delivery.md [hunk 2]
```diff
@@ -6,31 +6,31 @@ reviewed: 2017-03-02
...
 
-A level is created by declaring a topic exchange that is bound to a queue with a routing key of `1`, and is also bound to the exchange corresponding to `level - 1` with a routing key of `0`. The queue for the level is declared with an `x-message-ttl` value corresponding to `2^level` seconds. The queue is also declared with an `x-dead-letter-exchange` value corresponding to the `level - 1` exchange, so that when a message in the queue expires, it will be routed to the `level - 1` exchange.
+A delay level is created by declaring a topic exchange that is bound to a queue with a routing key of `1`, and to the exchange corresponding to `level - 1` with a routing key of `0`. The queue for the delay level is declared with an `x-message-ttl` value corresponding to `2^level` seconds. The queue is also declared with an `x-dead-letter-exchange` value corresponding to the `level - 1` exchange, so that when a message in the queue expires, it will be routed to the `level - 1` exchange.
```
Comment (SimonCropp) [inline nservicebus/rabbitmq/delayed-delivery.md]: @bording so https://www.rabbitmq.com/ttl.html is not relevant above. but should `x-message-ttl` link to it?

Comment (bording) [inline nservicebus/rabbitmq/delayed-delivery.md]: Sure, a link there could make sense, along with a link to https://www.rabbitmq.com/dlx.html for the `x-dead-letter-exchange` value.

Comment (bording) [inline nservicebus/rabbitmq/delayed-delivery.md]: I'll push up an update that has those.

File: nservicebus/rabbitmq/delayed-delivery.md (hunk 1 above)
Comment (indualagarsamy) [inline nservicebus/rabbitmq/delayed-delivery.md]: @SimonCropp - you got it. @bording - In my opinion, if we are using native rabbit, linking to documentation on the rabbitmq makes it better.

Comment (bording) [inline nservicebus/rabbitmq/delayed-delivery.md]: @indualagarsamy Sure, and we do that where it makes sense. I wasn't seeing a relevant link for the spot where you raised this comment though! 😁

**Pull request merged**